spec/javascript/ParkShow.js

```diff
@@ -0,0 +1,50 @@
+import ParkShow from '../../app/javascript/react/components/ParkShow'
+
+
+describe('ParkShow', () => {
+  let name,
```

Comment:
The original Enzyme article is misleading -- you don't actually need to define anything other than `wrapper` here. So, replace all this with just `let wrapper`